### PR TITLE
Brings back the best feature ever(Voices for spessmen)

### PR DESCRIPTION
**Pull request by Shadowlight213**
Repull of #16555

[meme](http://sam.wileycomputerworks.com/SS13/MEMES.wav)
:cl:
add: Spacemen now have voices
/:cl:
Voices for spessmen featuring voice acting done by Cheridan!

### CODE REVIEW

**Comment by kevinz000** [comment]
:3c

**Comment by evsey9** [comment]
bring back poo

**Comment by PKPenguin321** [comment]
If you're gonna add voices, [go all the way](https://github.com/jaggerestep/dectalk-ss13)

**Comment by evsey9** [comment]
OH SHIT THIS WAS TESTMERGED
THIS IS A FUCKING SHITTY PR
CLOSE IT

**Comment by Shadowlight213** [comment]
@PKPenguin321 OOC: Kevinz000: >everyone gets about 3 minutes of 5 second pings while they hear wgw in speech

File: code/modules/mob/living/carbon/human/say.dm [hunk 1]
```diff
@@ -10,9 +10,11 @@
 
 /mob/living/carbon/human/treat_message(message)
 	message = dna.species.handle_speech(message,src)
+	var/list/temp_message = splittext(message, " ") //List each word in the message
+	var/speechwords = min(round(temp_message.len), 5)
```
Comment (Giacom) [inline code/modules/mob/living/carbon/human/say.dm]: Why would you round the length of a list? That is guaranteed to be an unsigned integer.

Comment (Shadowlight213) [inline code/modules/mob/living/carbon/human/say.dm]: ask @Cheridan he coded it originally. :^)

Comment (optimumtact) [inline code/modules/mob/living/carbon/human/say.dm]: Sounds like a Cheri state of mind

**Comment by lzimann** [comment]
Can we let this die already ?

**Comment by optimumtact** [comment]
mememememeemememmeme?
mememeememem!

MemememememE!?!?!?

**Comment by Durkel** [comment]
Fucking memers

**Comment by ma44** [comment]
Not dank enough

**Comment by pudl** [comment]
its not funny when you repeat a meme someone else did for a second laugh. no effort memes, try next time

File: code/modules/mob/living/carbon/human/say.dm (hunk 1 above)
Comment (Cheridan) [inline code/modules/mob/living/carbon/human/say.dm]: u can never be too safe

**Comment by Alek2ander** [comment]
https://github.com/jaggerestep/dectalk-ss13

**Comment by Shadowlight213** [comment]
@PKPenguin321 @Alek2ander I actually looked into the possibility of doing that just now, but I can't add it.
As is written in the readme, that is just code for the wrapper. The text to speech libraries themselves are not contained in it and would need to be purchased.

**Comment by optimumtact** [comment]
We're probably not ever going to have text to speech for the simple fact it's annoying as shit

**Comment by Shadowlight213** [comment]
@optimumtact I thought that people were thinking about having text to speech replace vox?

**Pull request closed without merging**